### DISCUSSION TITLE
Possible to use surround to select/change/delete all text within * or **?

### DISCUSSION THREAD

**Discussion by sinewave** [esm7/obsidian-vimrc-support#76] [Q&A]
Hi, very new to vim and wondering if this plugin would allow me to create some mappings/commands that will select all the bolded/italic text. e.g. \****select all |this text**** similar to how cib or vib works with parentheses?

**Comment by esm7** [comment]
In "real" Vim sky is the limit, and while I'm not aware of a built-in way to treat the text between `*...*` as a text object, you can make it happen with some effort.
But CodeMirror (Obsidian's editor) is not real Vim, it implements just a very partial subset of Vim, and as of now I'm not aware of a way to do what you're asking.
It can be added as a feature request though, if there's enough demand and/or someone picks it up :)
